src/ToolBox/superpmi/superpmi/jitinstance.cpp

```diff
@@ -212,7 +212,8 @@ HRESULT JitInstance::StartUp(char*          PathToJit,
 
 bool JitInstance::reLoad(MethodContext* firstContext)
```

Comment:
Note, as a separate issue: nobody uses `reLoad` currently. It should really be refactored to share the similar/same code with `JitInstance::StartUp()`. In particular, reLoad doesn't do the GUID check, which it probably should, for safety.